docs/getting-started/istio/rollout.yaml

```diff
@@ -10,10 +10,13 @@ spec:
       stableService: rollouts-demo-stable
       trafficRouting:
         istio:
-          virtualService:
-            name: rollouts-demo-vsvc
+          virtualServices:
+          - name: rollouts-demo-vsvc1 # At least one virtualService is required
             routes:
             - primary # At least one route is required
+          - name: rollouts-demo-vsvc2
+            routes:
+            - secondary # At least one route is required
```

Comment:
Same here, can we add a new file named `rollout-multiple-vsvs.yaml`, instead of changing the existing one.

BTW, should this be primary, because there is no secondary route in `rollouts-demo-vsvc2`

Reply:
Updated the PR. Can you please check?